ci/gpu/build.sh

```diff
@@ -235,6 +235,11 @@ cd "$WORKSPACE/python/cudf/cudf"
 gpuci_logger "Python py.test for cuDF"
 py.test -n 8 --cache-clear --basetemp="$WORKSPACE/cudf-cuda-tmp" --ignore="$WORKSPACE/python/cudf/cudf/benchmarks" --junitxml="$WORKSPACE/junit-cudf.xml" -v --cov-config="$WORKSPACE/python/cudf/.coveragerc" --cov=cudf --cov-report=xml:"$WORKSPACE/python/cudf/cudf-coverage.xml" --cov-report term --dist=loadscope tests
 
+# Run benchmarks with both cudf and pandas to ensure compatibility is maintained.
```

Comment:
I'd put this after normal pytests and before notebook tests -- unless there's a reason that we can fail earlier with good error messages by running this before standard pytests.

I would also add in the comment that this only runs a _tiny_ benchmark input and not a full set of benchmarks.